tests/manage/pv_services/test_pvc_delete_subvolumegroup.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+import logging
+from ocs_ci.framework import config
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pvc import delete_pvcs
+from ocs_ci.ocs.resources.pod import get_ceph_tools_pod
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    skipif_ocs_version,
+    polarion_id,
+    bugzilla,
+    tier2,
+)
+from ocs_ci.utility.utils import TimeoutSampler
+
+logger = logging.getLogger(__name__)
+
+
+@skipif_ocs_version("<4.12")
+@polarion_id("OCS-4664")
+@bugzilla("2124469")
+@tier2
+class TestPvcDeleteSubVolumeGroup(ManageTest):
+    """
+    Delete PVC subvolumegroup and make sure PVC reaches bound state
+    """
+
+    vol_name = "ocs-storagecluster-cephfilesystem"
+    external_mode = False
+    if config.DEPLOYMENT["external_mode"]:
+        vol_name = "cephfs"
+        external_mode = True
```

Comment:
Is this `external_mode` being used somewhere and really needed?

Reply:
Removed